### ISSUE TITLE
Export PDF - selected directory based on last saved file and not on current one

### ISSUE THREAD

**Issue by schlamar** [MacDownApp/macdown#214]
If I save one *.md file, open another one and select "Export PDF", the directory of the last saved Markdown file is selected as export directory. Please default to the directory of the currently opened file, otherwise it is really annoying to work with and export multiple Markdown files in parallel.

**Comment by chrisozenne** [comment]
I agree, please default to the directory of the currently opened file. It would also be nice to have a command line interface for batch exporting markdown files into HTML or PDF.

**Comment by Jmuccigr** [comment]
Iʻm not sure that should be the default though. I often am exporting to a consistent place that isnʻt the folder of the current document.

How do other apps handle this?

**Comment by uranusjr** [comment]
I don’t use the export functionality in any apps much, but I think iWorks apps (Pages, Numbers, etc.) default to the same directory as the current file, as @chrisozenne described.

**Comment by Jmuccigr** [comment]
It doesn't work that way for me. Pages remembers the last export destination no matter which file is open and it shares that location with save. So if I save a new file to Desktop and then try an export, the export defaults to the Desktop too. That applies no matter where the open file is from. The same happens if I export and then save a new file; the save location is the same as the last export.

On Mavericks. Pages 5.2.2.

**Comment by netheril96** [comment]
Maybe it should be an option, whether the export location should default to the last or to the location of the markdown file. I prefer the location of the markdown file as I always pack them together.
